watcher/applicationprofilewatcher.go

```diff
@@ -19,7 +19,7 @@ import (
 
 // ApplicationProfileWatch watches and processes changes on ApplicationProfile resources
 func (wh *WatchHandler) ApplicationProfileWatch(ctx context.Context, workerPool *ants.PoolWithFunc) {
-	inputEvents := make(chan watch.Event)
+	eventQueue := NewCooldownQueue()
```

Comment:
question: is the default 5s enough for AP?

Reply:
I would increase it to 10s-15s to be on the safe side